### ISSUE TITLE
Colorize gem update issue

### ISSUE THREAD

**Issue by cernyjakub** [phallguy/shog#19]
colorize 1.0 has brought some cleanups and now I see these errors in log:

```
[ERROR] Could not log "start_processing.action_controller" event. NoMethodError: undefined method `magenta' for "===":String`
```

it seems to me they removed magenta color...

**Comment by PikachuEXE** [comment]
Seems fixed in 1.0.5
https://github.com/fazibear/colorize/blob/master/CHANGELOG.md#105--2023-06-22

**Comment by cernyjakub** [comment]
Perfect, OK now. Closing this issue.
